lib/improver/nbhood/vicinity.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've haven't moved this file for the moment, although I agree that this will probably want to be moved later, when it becomes clearer how we intend to group this type of functionality.